### ISSUE TITLE
Implement /TIME command

### ISSUE THREAD

**Issue by twostars** [srmeier/KnightOnline#101]
Command is:
```
9105	[Set Time] : /%s Hour Minute
```

Packet structure is (where the non-specified values are all 2 bytes each, as indicated by their names):
```
		uint16_t y, m, d;
		pkt >> y >> m >> d >> g_pMain->m_sHour >> g_pMain->m_sMin;
```

Note that the client only lets you set the hour and minute portion. It doesn't let you set the year/month/day: these remain 0, always. They're not used, so we don't need to supply anything for them.

When updating, we need to be sure to send the WIZ_TIME packet to all players currently logged in to sync their times as well.

So to summarise:

 - [x] Handle /TIME packet in client by parsing hour and minute e.g. "/time 12 30" and sending the WIZ_TIME packet to the server (this is already in place).
    - Note: The date portion is unused, so we simply leave those fields set to 0.
 - [x] Handle receiving WIZ_TIME to update the client's time (this is already in place).
 - [ ] Server needs to handle reading WIZ_TIME (from GMs only) to set the server's hour/minute (not date -- it's unused, so we can ignore it) and update all currently online players with the WIZ_TIME packet.
